src/components/Header.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-  <div id="navbar" class="navbar">
-    <div class="nav">
+  <div id="header" class="header">
+    <div class="header-wrapper">
       <div class="logo-container">
-        <img class="logo" alt="Mesh logo" src="../assets/mesh-logo.png">
+        <router-link to="/">
+          <img class="logo" alt="Mesh logo" src="../assets/mesh-logo.png">
+        </router-link>
       </div>
       <div class="link-container">
         <span class="date">
           <img class="calendar" alt="Calendar icon" src="../assets/calendar.svg">
           {{ date }}
         </span> |
-        <span class="name">Hi, Nikki</span> |
-        <router-link to="/">Home</router-link> |
-        <router-link to="/about">About</router-link>
+        <span class="name">Hi, Nikki</span>
```

Comment:
Changed navbar to header so we don't get it confused with an actual navigation bar